packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,7 +5,7 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.9
+    tag: v1.0.10-rc1
```

Comment:
can be updated to v1.0.10

Reply:
I have updated the cis-operator tag to the latest which is 1.0.11-rc1